### PR TITLE
Details of rewards including rewards split

### PR DESCRIPTION
**Pull request by sennui**

### CODE REVIEW

File: consensus/rewards.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Miners' rewards
+
+Aeternity is Proof of Work blockchain. This is a description of rewards paid to miners.
+
+## Introduction
+
+Miners mine key blocks and receive rewards composed out of two ingredients:
```
Comment (lucafavatella) [inline consensus/rewards.md]: This seems already described in [bitcoin-ng.md](https://github.com/aeternity/protocol/blame/c638d7728380bfdf85968dba1d22672f49664727/consensus/bitcoin-ng.md#L39_L46).

File: consensus/rewards.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+1. Coinbase-like reward for mining a key block
+2. Fees and gas paid by users for transactions and contract calls
+
+There is also special case altering rewarding scheme. It crypto economic incentive designed to lower risk of fraud - and it is called Proof of Fraud.
```
Comment (lucafavatella) [inline consensus/rewards.md]: This seems already described in [bitcoin-ng.md](https://github.com/aeternity/protocol/blame/c638d7728380bfdf85968dba1d22672f49664727/consensus/bitcoin-ng.md#L86).

File: consensus/rewards.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Miners rewards are also affected by protocol protected mechanism of reward split.
+
+### Reward for mining a key block
+
+Miners include beneficiary address in a key block. The reward is sent to it with protocol protected delay.
```
Comment (lucafavatella) [inline consensus/rewards.md]: This seems already mostly described in [bitcoin-ng.md](https://github.com/aeternity/protocol/blame/c638d7728380bfdf85968dba1d22672f49664727/consensus/bitcoin-ng.md#L48).

File: consensus/rewards.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+The size of the reward is computed based on inflation curve
+
+### Fees and gas
+
+Users need to pay for execution of their actions. According to Bitcoin-NG protocol a miner gets 60% of previous generation fees and 40% of the next one.
```
Comment (lucafavatella) [inline consensus/rewards.md]: This seems already described in [bitcoin-ng.md](https://github.com/aeternity/protocol/blame/c638d7728380bfdf85968dba1d22672f49664727/consensus/bitcoin-ng.md#L39-L46).

File: consensus/rewards.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+### Proof of Fraud
+
+If a miner decides to cheat and double spend by creating a fork within generation of his leadership PoF is a mechanism to detect it and punish. Such a fork will be resolved by usual Proof of Work fork algorithm and no funds are double spent. Furthermore, the next leader is able to signal malicious activity by including two microblock headers with the same parent and the same leader. The consequences are following:
+1. Malicious leader is not rewarded with mining reward
```
Comment (lucafavatella) [inline consensus/rewards.md]: This seems already described in [bitcoin-ng.md](https://github.com/aeternity/protocol/blame/c638d7728380bfdf85968dba1d22672f49664727/consensus/bitcoin-ng.md#L133-L135).

File: consensus/rewards.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+2. The signaling miner receives 5% of according reward
```
Comment (lucafavatella) [inline consensus/rewards.md]: [bitcoin-ng.md](https://github.com/aeternity/protocol/blame/c638d7728380bfdf85968dba1d22672f49664727/consensus/bitcoin-ng.md#L136) makes more clear that this portion is only of coinbase - i.e. not of all of the line above. (Though does not detail 5%.)

**Comment by lucafavatella** [comment]
> May I suggest refining bitcoin-ng.md rather than creating new file? I am concerned of docs describing mostly same thing become misaligned.

See #350 for my counterproposal.

**Comment by thepiwo** [comment]
@lucafavatella which of these will be merged, I want to add the protocol BRI % there as well

**Comment by lucafavatella** [comment]
> @lucafavatella which of these will be merged, I want to add the protocol BRI % there as well

@thepiwo This is now closed, please prefer #350

**Pull request closed without merging**